builder/googlecompute/step_create_image.go

```diff
@@ -22,6 +22,19 @@ func (s *StepCreateImage) Run(state multistep.StateBag) multistep.StepAction {
 	driver := state.Get("driver").(Driver)
 	ui := state.Get("ui").(packer.Ui)
 
+	if config.PackerForce && config.imageAlreadyExists {
```

Comment:
I might be missing something, but it looks to me like this will never run, since `imageAlreadyExists` is only set if `PackerForce` is false, so if `PackerForce` is true, `imageAlreadyExists` will be nil, and the if statement will evaluate to false